.github/workflows/validate-new-issue.yml

```diff
@@ -9,7 +9,22 @@ jobs:
     - uses: actions/checkout@v2
     - name: "Validate issue against templates"
       shell: bash
+      env:
+        GITHUB_TOKEN: ${{ github.token }}
       run: |
+        # Trust users who belong to the getsentry org (id: 1396951). Due to API
+        # limitations, we can only see this if the user's membership is public.
+        sentry_org=$(
+          gh api "https://api.github.com/users/${{ github.actor }}/orgs" \
+            | jq '.[] | select(.id==1396951)'
```

Comment:
This seems easier and more suitable: https://docs.github.com/en/free-pro-team@latest/rest/reference/orgs#check-organization-membership-for-a-user

`https://api.github.com/orgs/getsentry/members/chadwhitacre`

Reply:
I'll try it out but I think we won't have perms:

> To see many of the organization response values, you need to be an authenticated organization owner with the admin:org scope.

https://docs.github.com/en/free-pro-team@latest/rest/reference/orgs#get-an-organization

Reply:
I don't seem to be able to get anything from the org endpoint:

<img width="836" alt="Screen Shot 2021-01-15 at 9 27 51 AM" src="https://user-images.githubusercontent.com/134455/104738851-0af93a00-5714-11eb-9550-4f5477297d83.png">

```diff
diff --git a/.github/workflows/validate-new-issue.yml b/.github/workflows/validate-new-issue.yml
index a5f49c8..12f6be6 100644
--- a/.github/workflows/validate-new-issue.yml
+++ b/.github/workflows/validate-new-issue.yml
@@ -12,9 +12,17 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ github.token }}
       run: |
+        echo  "Org endpoint shows nothing:"
         gh api "https://api.github.com/orgs/getsentry/members/${{ github.actor }}"
+
+        echo  ""
+        echo  ""
+        echo  "User endpoint shows something:"
         gh api "https://api.github.com/users/${{ github.actor }}/orgs"
 
+        echo  ""
+        echo  ""
+
         # Trust users who belong to the getsentry org (id: 1396951). Due to API
         # limitations, we can only see this if the user's membership is public.
         sentry_org=$(
```

Fwiw, curling the endpoint anonymously shows that it redirects to a public endpoint.

```
$ curl -I "https://api.github.com/orgs/getsentry/members/chadwhitacre"
HTTP/1.1 302 Found
Server: GitHub.com
Date: Fri, 15 Jan 2021 14:24:54 GMT
Content-Type: text/html;charset=utf-8
Content-Length: 0
Status: 302 Found
Location: https://api.github.com/organizations/1396951/public_members/chadwhitacre
```

Reply:
I'm going to stick with the user endpoint. It at least provides some way for staff to dodge this restriction. If someone is really opposed to publicizing their Sentry membership, it's pretty easy to fool the bot and then update the issue. 😁

Reply:
If you do `curl -vL` you'd see that the final URL gives you a `204` which means there _is_ a user member of that org. If there isn't any, it gets you a `404`.

Up to you tho.

Reply:
I was able to get this to work using `gh api`. It does what we want! The GHA user is able to see private memberships in the `getsentry` org. 👍